.travis.yml

```diff
@@ -30,6 +30,15 @@ jobs:
     - mkdir -p $DIST_DIR_S3 && cp $BUILD_DIR/$ZIP_FILENAME $DIST_DIR_S3/$ZIP_FILENAME
     - mkdir -p $DIST_DIR_GITHUB && cp $BUILD_DIR/$ZIP_FILENAME $DIST_DIR_GITHUB/$GITHUB_RELEASE_FILENAME
     deploy:
+      - provider: npm
+        email: pjaudiomv@gmail.com
```

Comment:
maybe we should use the service account?

Reply:
can we invite fortyseven like we do for wordpress plugins?

Reply:
yea wasnt sure. whats that email

Reply:
ah yes ok< I gotcha

Reply:
I think you have to invite him, use admin bmlt email